src/platform/unix/linux/mod.rs

```diff
@@ -26,18 +26,64 @@ use std::thread;
 
 const MAX_FDS_IN_CMSG: u32 = 64;
 
+const SCM_RIGHTS: c_int = 0x01;
```

Comment:
`libc` I'd say. `libc` is for the raw FFI interfaces; while `nix` is for safe wrappers around them. Using `nix` to abstract all these dirty details would be nice to have, but it's a second (much bigger) step...